RubberduckTests/Inspections/EmptyIfBlockInspectionTests.cs

```diff
@@ -369,39 +369,6 @@ Sub Bar()
 
         [TestMethod]
         [TestCategory("Inspections")]
-        public void EmptyIfBlock_QuickFixRemovesSingleLineIf_NoWhiteSpace()
-        {
-            const string inputCode =
-@"Sub Foo()
-    If True Then Else(Bar)
-End Sub
-
-Sub Bar()
-End Sub";
-
-            const string expectedCode =
-@"Sub Foo()
-    If Not True Then (Bar)
-End Sub
-
-Sub Bar()
-End Sub";
-
```

Comment:
This test has been removed because it is illegal in VBA to enclose a call statement in parentheses. This is enforced by the VBE and, now, the grammar reflects this, too.